.stylelintrc

```diff
@@ -3,5 +3,13 @@
   "extends": [
     "stylelint-config-recommended",
     "stylelint-config-styled-components"
-  ]
+  ],
+  "rules": {
+    "no-descending-specificity": null,
+    "property-no-vendor-prefix": [
```

Comment:
why add these two rules?

Reply:
1. We have ~30 `no-descending-specificity`'s errors, and I don't think it's a really important rule for me. For example, it will have an error for below:
```
div {
  &:disable {
     // ...
     &:hover {
       // ...
      }
  }
  &:hover {
    // ...
  }
}

```
There is no need to move `&:hover  before `&:disable` for me, I don't think it has a big impact on maintenance.
2. `property-no-vendor-prefix` is used to ignore the error for `-webkit-appearance` css attribute.

Reply:
for 2, why we need `-webkit-appearance` in our css code? I thought `styled-components` can prefix it for us?

Reply:
Oh yes `styled-components` should do automatic vendor prefixing for us. Will update it, thanks!